doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -5499,90 +5497,90 @@ Natural number
 .. prodn:: nat_or_ident ::= {| @natural | @ident }
 
 where :token:`ident` is an Ltac variable denoting a standard Coq number
-(should not be the name of a tactic which can be followed by a
-bracket ``[``, like ``do``, ``have``,…)
+(should not be the name of a tactic that can be followed by a
+bracket ``[``, such as ``do``, ``have``,…)
 
 Items and switches
 ~~~~~~~~~~~~~~~~~~
 
 .. prodn:: ssr_binder ::= {| @ident | ( @ident {? : @term } ) }
 
-binder see :ref:`abbreviations_ssr`.
+binder (see :ref:`abbreviations_ssr`)
 
 .. prodn:: clear_switch ::= { {+ @ident } }
 
-clear switch see :ref:`discharge_ssr`
+clear switch (see :ref:`discharge_ssr`)
 
 .. prodn:: c_pattern ::= {? {| @term in | @term as } } @ident in @term
 
-context pattern see :ref:`contextual_patterns_ssr`
+context pattern (see :ref:`contextual_patterns_ssr`)
 
 .. prodn:: d_item ::= {? {| @occ_switch | @clear_switch } } {? {| @term | ( @c_pattern ) } }
 
-discharge item see :ref:`discharge_ssr`
+discharge item (see :ref:`discharge_ssr`)
 
 .. prodn:: gen_item ::= {| {? @ } @ident | ( @ident ) | ( {? @ } @ident := @c_pattern ) }
 
-generalization item see :ref:`structure_ssr`
+generalization item (see :ref:`structure_ssr`)
 
 .. prodn:: i_pattern ::= {| @ident | > | _ | ? | * | + | {? @occ_switch } {| -> | <- } | [ {?| @i_item } ] | - | [: {+ @ident } ] }
 
-intro pattern :ref:`introduction_ssr`
+intro pattern (see :ref:`introduction_ssr`)
 
 .. prodn:: i_item ::= {| @clear_switch | @s_item | @i_pattern | @i_view | @i_block }
 
-view :ref:`introduction_ssr`
+view (see :ref:`introduction_ssr`)
 
 .. prodn::
    i_view ::= {? %{%} } {| /@term | /ltac:( @tactic ) }
 
-intro block :ref:`introduction_ssr`
+intro block (see :ref:`introduction_ssr`)
 
 .. prodn::
    i_block ::= {| [^ @ident ] | [^~ {| @ident | @natural } ] }
 
-intro item  see :ref:`introduction_ssr`
+intro item (see :ref:`introduction_ssr`)
 
 .. prodn:: int_mult ::= {? @natural } @mult_mark
 
-multiplier  see :ref:`iteration_ssr`
+multiplier (see :ref:`iteration_ssr`)
 
 .. prodn:: occ_switch ::= { {? {| + | - } } {* @natural } }
 
-occur. switch see :ref:`occurrence_selection_ssr`
+occur. switch (see :ref:`occurrence_selection_ssr`)
 
 .. prodn:: mult ::= {? @natural } @mult_mark
 
-multiplier see :ref:`iteration_ssr`
+multiplier (see :ref:`iteration_ssr`)
 
 .. prodn:: mult_mark ::= {| ? | ! }
 
-multiplier mark see :ref:`iteration_ssr`
+multiplier, mar (see :ref:`iteration_ssr`)
```

Comment:
What is "mar"?

Reply:
Typo on my part. Fixed in upcoming updated PR.